workflows/createtaxdb.nf

```diff
@@ -99,22 +133,27 @@ workflow CREATETAXDB {
             unzipped: true
         }
 
-        GUNZIP_AA(ch_aa_for_unzipping.zipped)
-        ch_prepped_aa_fastas_ungrouped = GUNZIP_AA.out.gunzip.mix(ch_aa_for_unzipping.unzipped)
+        ch_aa_for_unzipping.zipped
```

Comment:
I think the AA stuff looks a bit out of sync with the DNA stuff, can you confirm?

e.g. the collate here is still has `set` and is missing some `maps` the DNA equivalent hsa